apps/src/codebridge/FileBrowser/FileBrowser.tsx

```diff
@@ -65,13 +66,7 @@ export const FileBrowser = React.memo(() => {
     }),
     useSensor(KeyboardSensor, {
       coordinateGetter: fileBrowserKeyboardCoordinateGetter(projectFolders),
-      keyboardCodes: {
-        // Start dragging on 'm', so 'enter' and 'space' can be used to open/close a folder.
-        // TODO: expose a menu to users of our keyboard options, until then this is a hidden feature.
-        start: ['KeyM'],
-        cancel: ['Escape'],
-        end: ['KeyM', 'Enter', 'Space'],
```

Comment:
Noticing that 'Enter' and 'Space' are removed. Is it possible to add back?

Reply:
added back!